tests/ComInterfaceAttributes_Test.cs

```diff
@@ -18,23 +18,26 @@ public void HasGuids([ValueSource("GetAllComVisibleTypes")] Type type)
 		}
 
 		[Test]
-		public void InterfaceType([ValueSource("GetComVisibleInterfaceTypes")] Type type)
+		public void InterfaceType([ValueSource("GetComVisibleInterfaceTypes")]
+			Type type)
 		{
 			var interfaceTypeAttribute = type.GetCustomAttribute<InterfaceTypeAttribute>();
 			if (interfaceTypeAttribute != null)
 				Assert.That(interfaceTypeAttribute.Value, Is.EqualTo(ComInterfaceType.InterfaceIsDual));
 		}
 
 		[Test]
-		public void ClassInterface([ValueSource("GetComVisibleClassTypes")] Type type)
+		public void ClassInterface([ValueSource("GetComVisibleClassTypes")]
+			Type type)
```

Comment:
I think there is unnecessary edits in this file

Reply:
Reverted changes